packages/block-editor/src/store/private-selectors.js

```diff
@@ -550,3 +550,13 @@ export function isZoomOutMode( state ) {
 export function getSectionRootClientId( state ) {
 	return state.settings?.[ sectionRootClientIdKey ];
 }
+
+/**
+ * Where the inserter should insert into.
+ *
+ * @param {Object} state
+ * @return {Object} Of where the insertion point in the block editor is or null if none is set.
+ */
+export function getInserterInsertionPoint( state ) {
```

Comment:
Could we name this `getInsertionPoint` instead? The same name is in the editor package. I would much prefer `getInsertionPoint` as it could be set without the inserter and we don't really want the block-editor to know about the editor package.

Reply:
If there's no name clash, then `set/getInsertionPoint` sounds much better.

Reply:
There isn't a name clash if I change enough names 😉 😅 I changed quite a bit of naming in https://github.com/WordPress/gutenberg/pull/65098/commits/64dcd1fc99efe108ccf3564dd5e178b71daa8eb1. The main source of confusion for it will be the naming of `getBlockInsertionPoint`, but at least now when you look at it it's more obvious that it's returning the state of `insertionCue`. So, while still confusing, maybe it'll be easier to understand the flow.